osbs/http.py

```diff
@@ -34,6 +41,22 @@
 logger = logging.getLogger(__name__)
 
 
+class OSBSRetry(Retry):
+    def is_forced_retry(self, method, status_code, has_retry_after=False):
```

Comment:
This happens inside urrlib3: whenever the request is finished it is evaluating if retry is required: https://github.com/requests/requests/blob/v2.6.0/requests/packages/urllib3/connectionpool.py#L640